.gitlab-ci.yml

```diff
@@ -414,28 +416,23 @@ test:prep:
 
 test:staging-deployment:chrome:
   extends: .template:test:staging-deployment
-  resource_group: test-staging-deployment-chrome
   script:
     - npx playwright install chromium
     - npm run test -- --project=chromium
   start_in: 15 minutes
```

Comment:
instead of the sleep in the helm bump, we could cut this down to 3 minutes and stagger the other jobs by a little, to ensure the order is still kept as the jobs are supposed to be run in order of browser popularity

Reply:
Something like this, if I understood it correctly?